### PR TITLE
LPS-145363 [CP 2.0] Allow users to remove additional admin during DXP Cloud setup

### PR DESCRIPTION
**Pull request by joaosvassis**
### How it was developed?

- Added a new button using Clay to remove additional admin, and created a logic to hide the remove button if there is only 1 admin added, and if there are 20 admin added, the add button is hidden.

### CODE REVIEW

**Review by Lucas2691 — APPROVED** [review]
Good job.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/index.js [hunk 1]
```diff
@@ -158,79 +162,107 @@ const SetupDXPCloudPage = ({
...
-							{project.name.length > 71
-								? project.name.substring(0, 71) + '...'
-								: project.name}
-						</strong>
-					</p>
-				</div>
+								<p className="dxp-cloud-project-name text-neutral-6 text-paragraph-lg">
+									<strong>
+										{project.name.length > 71
+											? project.name.substring(0, 71) +
```
Comment (igordmf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/index.js:176]: I think would be better use constants instead magic numbers

https://pt.wikipedia.org/wiki/N%C3%BAmero_m%C3%A1gico_(programa%C3%A7%C3%A3o_de_sistemas)

https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

**Review by igordmf — COMMENTED** [review]
Good job!

**Review by claricebrasil — APPROVED** [review]
Great work, Joao! Congratulations!

**Review by igordmf — APPROVED** [review]
Good job!

**Review by joanagrego — APPROVED** [review]
Nice job!

**Review by thaynam — APPROVED** [review]

**Comment by iagochaves** [comment]
Just starting reviewing :)

**Comment by iagochaves** [comment]
re-sent https://github.com/pt-liferay-solutions/liferay-portal/pull/996

**Pull request closed without merging**